modules/search/instant-search/components/search-result-minimal.jsx

```diff
@@ -114,7 +114,7 @@ class SearchResultMinimal extends Component {
 		}
 		const noMatchingContent = ! highlight.content || highlight.content[ 0 ] === '';
 		return (
-			<div className="jetpack-instant-search__result-minimal">
+			<div className="jetpack-instant-search__result-minimal" role="listitem">
```

Comment:
Could we use an `<ul>` for search results instead?

Reply:
We'd need to change its parent from `main` to `ul`, so I held off on making that change. 

If we're talking semantics, I think `ol` makes more sense. We'd need to add a bit of custom styling to ensure that the browser's default list styling doesn't kick in.

Reply:
I figured out a way to use an ordered list for the search results.